Go run main.go won't work

```
go run main.go
# command-line-arguments
./main.go:29:10: undefined: NewAuth
./main.go:30:13: undefined: NewWebhook
./main.go:32:34: undefined: NewBuildServer
```

use

    go run ./...

--- a/README.md
+++ b/README.md
@@ -45,7 +45,7 @@ SCALEKIT_CLIENT_SECRET = client_secret
 
 ```sh
 # Run the server:
-go run main.go
+go run ./...
 ```
 
 ```sh

